add new srv SpawnEntities and related msg, for ros1

tetorea opened the pull request.

Commit:
add new srv SpawnEntities and related msg, for ros1

Tadinu commented on msg/SpawnEntityState.msg:
```diff
@@ -0,0 +1,4 @@
+# Holds an entity state, xml description and UE tags
+string xml                  # Entity XML description as a string : entity type, etc
+ue_msgs/EntityState state   # Entity pose, twist, name, reference_frame
+string[] tags               # tags to use in UE
```
Just a comment, `Spawn` prefix makes itself signified as a service/action, so given the msg contents, could it be named like `EntityInfo` for clarity?

tetorea replied:
Yeah maybe `EntityInfo` is a better name

yuokamoto commented on msg/SpawnEntityState.msg:
```diff
@@ -0,0 +1,4 @@
+# Holds an entity state, xml description and UE tags
+string xml                  # Entity XML description as a string : entity type, etc
```
Do we need xml?

Tadinu replied:
I suppose we could rename it to `ModelName` for its being used as [entity key name](https://github.com/rapyuta-robotics/RapyutaSimulationPlugins/blob/devel/Source/RapyutaSimulationPlugins/Private/Tools/SimulationState.cpp#L256) which essentially represents an object/robot's ModelName.
Also, since our Sim now only requires users to only provide a path to their own master folder of URDF/SDF models, &  automatically load their contents under the hood, thus they would not have to pass the bulky XML content in each service call's message.

tetorea replied:
I added a new message so that the service uses an array of this new type, especially as the message has an array for tags.  

The xml variable in SpawnEntityState.msg is like the one used in SpawnEntity service. Actually it's the type of object to spawn without any xml formatting so I could either change the variable name, or change its processing in the service in order to use xml formatting.  

The SpawnEntities service also has an xml variable which is not used for now but I added it in case it could be useful afterwards

Tadinu replied:
`SpawnEntities`'s xml is quite a legacy name, when we intended to pass the whole xml content through the message I guess. So it should be also renamed for clarity in the handling on Sim side, since our Sim should do the xml loading itself as already did, allowing users to only need to pass the Entity Model name only, which is also more economic.

tetorea replied:
I decided to simplify the service in order to remove the new msg.
There are some bug/limitations in the code generator, for example, the `tags` string array was "translated" as a simple string variable.
In this case, the need for a new message is less obvious.

So now, each entity is defined by a type, state and tag.

yuokamoto reviewed (commented):
Do we need both topic and service interface?

Commit:
modify SpawnEntities service, remove new msg

- remove SpawnEntityState.msg. It was used to attach a string[] tags variable to each entity in a list but the code generator actually process it as a string variable, for now
As it is used only for UE tags variable and not useful for now, it is simpler to just remove it
It means, there can only be one tag for each entity, for now. (or we need to define a delimiter inside the tags string to allow more tags : json, xml, special character ...)

- fix variable name and use different arrays instead of an array of structure, for the service SpawnEntities
Makes the life easier for current code generator and remove the need for a new ms

Commit:
reset readme, add changelog with updated information
